cedar-lean/Cedar/Thm/Validation/Typechecker/Call.lean

```diff
@@ -373,6 +373,46 @@ theorem type_of_call_ipAddr_recognizer_is_sound {xfn : ExtFun} {xs : List Expr}
       apply bool_is_instance_of_anyBool
     }
 
+-- It's impossible to call `unknown` in a well typed way
+theorem type_of_unknown_call_untyped :
+  ∀ (f : ExtFun) (xs : List Expr) (c : Capabilities) (env : Environment),
```

Comment:
Our style would usually be
```suggestion
theorem type_of_unknown_call_untyped (f : ExtFun) (xs : List Expr) (c : Capabilities) (env : Environment) :
```

I'm not sure offhand why/whether we should prefer one style over the other